Makefile

```
@@ -71,6 +71,25 @@ image-%:
 	$(eval img := $(subst image-,,$@))
 	gcloud builds submit --config './images/$(img)/cloudbuild.yaml' './images/$(img)'
 
+RUNTIME := podman
```

I would argue that most people use `docker`, and that this won't change in a while. Therefor I suggest:
```suggestion
RUNTIME ?= docker
```

This should allow you to change the to a different runtime on your end.

---

Yep :+1: